client/src/i18n/en-US.json

```diff
@@ -794,42 +794,42 @@
     "list": "List of Notifications",
     "review": {
       "requested": {
-        "short": "{user_username} has requested your review on {object_name}"
+        "short": "{user_username} has requested your review on {object_name}."
       },
       "commentReplied": {
-        "short": "{user_username} has replied to your comment on {object_name}"
+        "short": "{user_username} has replied to your comment on {object_name}."
       }
     },
     "submission": {
       "invitation": {
         "review_coordinator": {
           "accepted": {
-            "short": "{data_invitee_display_label} has accepted the invitation from {data_inviter_display_label} to coordinate the review of {data_submission_title}"
+            "short": "{data_invitee_display_label} has accepted the invitation from {data_inviter_display_label} to review coordinate {data_submission_title}."
```

Comment:
"review coordinate" gives me pause when I read it. I imagine new users unfamiliar with peer review will find it more awkward-sounding and unclear than "coordinate the review of". I'm willing to let this edit remain but I wanted to note my apprehension with it.

Reply:
Review coordinate should remain. I do want to point out that here you write "coordinate the review of" and on another page "coordinate reviewers for." These are not the same thing, and the former is a more accurate description of what review coordinators do. (In publishing, editors also "coordinate reviewers".) Review coordinate is also grammatically more concise.

Reply:
I'm not able to find an instance of "coordinate reviewers for" appearing in the application. Can you provide a screenshot of where this appears? 

Also, in the interest of using plain and accessible language, I would encourage looking at our copyediting with a preference for clarity over brevity. I know being brief is often more clear but there are occasions where wordiness is more clear and I believe this is arguably one of those occasions.

Reply:
It is likely not appearing because I edited it as well. It's line 368. 

"Review coordinate" should remain. It's grammatically more concise, clearer, and a more accurate description of the role. If we need to discuss this more, we can in a meeting.

Reply:
![Screenshot 2023-05-18 at 14 49 38](https://github.com/MESH-Research/Pilcrow/assets/132350717/9e76ef2d-61e2-493a-93bb-872ac943c306)